### PR TITLE
Journey steps

### PR DESCRIPTION
**Pull request by virginiacc**
## Additions
- Adds steps 2-5 of journey

## Changes
- Styling tweaks for journey pages
- Decreases width of journey page wrapper to match consumerfinance.gov page width

## Screenshots
screen shot 2015-03-02 at 1 24 23 am

### CODE REVIEW

File: src/process/maintain/index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            "Learn how to wisely manage your loan.",
+            "Understand when you should refinance.",
+            "Know when to get help."
+        ],
+        "actions": [
+            "Make regular mortgage payments",
+            "Manage your spending carefully",
+            "Consider ways to pay off your home sooner",
+            "If you have an adjustable-rate mortgage, know when the rate will change",
+            "Assess whether to refine",
```
Comment (cfarm) [inline src/process/maintain/index.html]: Looks like a typo, should say "Assess whether to refinance"

File: src/process/prepare/index.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            "Determine how much you can spend on a home",
+            "Decide if it's the right time to buy a new home",
+            "Gather your paperwork"
+        ],
+        "actions": [
+            "Clarify your goals",
+            "Check your credit",
+            "Assess your spending",
+            "Budget for new or changed expenses",
+            "Determine your downpayment",
```
Comment (cfarm) [inline src/process/prepare/index.html]: should be "down payment" ?

**Comment by cfarm** [comment]
A couple of typos for the content, otherwise the code looks good.

We should talk about whitespace in our tabs at some point... :scream_cat:

File: src/process/maintain/index.html (hunk 1 above)
Comment (virginiacc) [inline src/process/maintain/index.html]: That would make more sense. ;)

**Comment by virginiacc** [comment]
The typos should be fixed -- thanks for catching those!

**Pull request merged**